Website/Providers/DataProviders/SqlDataProvider/07.04.00.SqlDataProvider

```diff
@@ -691,7 +695,7 @@ BEGIN
         'Reject',
         'Reject',
         NULL,
-        1,
+        2,
```

Comment:
Breaking Turbo scripts is not a valid reason for altering the behavior of DNN. If there is an index that you feel should be added to the core then submit the change and it can be reviewed and approved. Until that index is an accepted part of the standard DNN platform it should not be part of deliberation process for accepting or rejecting additional changes.

Just based on looking at other NotificationTypeActions it does appear that the correct value should be a 3 (not a 1 or a 2).  I'll check with the original developers of the notification feature to determine the correct behavior.